src/Game.cpp

```diff
@@ -15,3 +15,18 @@ int Game::score() const
     //needs implementation after made checkValueFrame()
     return 0;
 }
+
+int Game::checkValue(const std::string & value)
+{
+    int value_;
+    if ((value[0] == 'X') || (value[1] == '/'))
+    {
+        value_ = 10;
+    }
+    else
+    {
+        value_= ((value[0] - '0')+(value[1] - '0'));
```

Comment:
@anorberciak you can write function asciiToDec, the code will be cleaner

Reply:
https://en.cppreference.com/w/cpp/string/basic_string/stol

Reply:
thx @LordLukin and @Tomek52 , I just finished it :)